sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -305,6 +316,22 @@ export interface PartitionProperties {
     partitionId: string;
 }
 
+// @public
+export interface PartitionPublishingOptions {
+    ownerLevel?: number;
+    producerGroupId?: number;
```

Comment:
it could be confusing when some IDs are numbers but others are strings, I wonder if this inconsistency necessary

Reply:
It's a number that is passed to "com.microsoft:producer-id" in the amqp message. Probably not directly created by end users.

Reply:
no longer exposed.